### ISSUE TITLE
bluetooth not found - running watchmate from a virtualbox on an old macbook

### ISSUE THREAD

**Issue by luto65** [azymohliad/watchmate#66]
Hi, i have an old Mac (2013) with Big Slur (OS 11.7.10) .
I installed Oracle VirtualBox and in that i loaded Open Suse.
in Open Suse i installed watchmate.
When I launch it, does not find the bluetooth.
Here the log

F: Opening user flatpak installation at path /home/luca/.local/share/flatpak
F: Opening system flatpak installation at path /var/lib/flatpak
F: Opening user flatpak installation at path /home/luca/.local/share/flatpak
F: Opening system flatpak installation at path /var/lib/flatpak
F: /var/lib/flatpak/runtime/org.gnome.Platform/x86_64/45/730b4b38a4a285f7a8c995571d2b1d65414e5fb4d5dcdca6ac20cd7635960238/files/lib32 does not exist
F: Cleaning up unused container id 1697912969
F: Cleaning up per-app-ID state for io.gitlab.azymohliad.WatchMate
F: Allocated instance id 2884749162
F: Add defaults in dir /io/gitlab/azymohliad/WatchMate/
F: Add locks in dir /io/gitlab/azymohliad/WatchMate/
F: Allowing dri access
F: Allowing wayland access
F: Allowing x11 access
F: Running '/usr/bin/bwrap --args 37 -- /usr/bin/xdg-dbus-proxy --args=40'
F: Running '/usr/bin/bwrap --args 37 -- watchmate'
libEGL warning: DRI2: failed to authenticate
MESA: error: ZINK: failed to choose pdev
libEGL warning: egl: failed to create dri2 screen

(watchmate:2): Gtk-WARNING **: 08:59:53.717: Unable to acquire the address of the accessibility bus: GDBus.Error:org.freedesktop.DBus.Error.ServiceUnknown: org.freedesktop.DBus.Error.ServiceUnknown. If you are attempting to run GTK without a11y support, GTK_A11Y should be set to 'none'.
[ERROR watchmate::ui::devices_page] Failed to initialize bluetooth adapter: internal error: D-Bus error org.freedesktop.DBus.Error.TimedOut: Failed to activate service 'org.bluez': timed out (service_start_timeout=25000ms)

while I do not think this be an issue with your code, perhaps could be useful for others that are trying to execute the same from a virtual machine to know how to fix that..

**Comment by azymohliad** [comment]
Yeah, it is outside of the application control. The only log coming directly from Watchmate code here is this:
```
[ERROR watchmate::ui::devices_page] Failed to initialize bluetooth adapter: internal error: D-Bus error
```
in response to [bluer::Session::default_adapter()](https://docs.rs/bluer/latest/bluer/struct.Session.html#method.default_adapter) error, and there's nothing it can do about it.

Does the bluetooth work for other apps in your VM? Is the adapter forwarded there from host OS? Sorry for a bit presumptuous question, but just to be sure.
